src/PushPackage.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\Deploy;
+
+use DigipolisGent\Robo\Task\Deploy\Scp\Factory\ScpFactoryInterface;
+use DigipolisGent\Robo\Task\Deploy\Scp\Factory\ScpPhpseclibFactory;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Factory\SshFactoryInterface;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Factory\SshPhpseclibFactory;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Auth\AbstractAuth;
+use InvalidArgumentException;
+use Robo\Contract\BuilderAwareInterface;
+use Robo\Task\BaseTask;
+
+class PushPackage extends BaseTask implements BuilderAwareInterface
+{
+    use loadTasks;
+    use \Robo\Task\Remote\loadTasks;
+    use \Robo\TaskAccessor;
+
+    /**
+     * The server to scp to/from.
```

Comment:
Yes, in the separate files, but this class only _pushes_ the package to the server AFAICS